src/glyph.ts

```diff
@@ -344,16 +355,22 @@ export class Glyph extends Element {
           bboxComp.addPoint(penX, penY);
           break;
         case OutlineCode.QUADRATIC:
+          x1 = go.nextX();
+          y1 = go.nextY();
           x = go.nextX();
           y = go.nextY();
-          bboxComp.addQuadraticCurve(penX, penY, go.nextX(), go.nextY(), x, y);
+          bboxComp.addQuadraticCurve(penX, penY, x1, y1, x, y);
```

Comment:
I have a branch that changes `BoundingBoxComputation`'s API to match the drawing API, i.e. `moveTo`, `bezierCurveTo`, etc. I can send a pull request with just that change once this PR is submitted if you like: https://github.com/tommadams/vexflow/blob/svg/src/boundingboxcomputation.ts

Reply:
Can bounding boxes be precomputed? When I extract the paths via opentype.js, I can also access `glyph.getBoundingBox()`, which provides `xMin`, `yMin`, `xMax`, `yMax`.

Assuming their bounding box calculation is accurate, can we just scale and translate the bounding box for our purposes?

Reply:
The BoundingBoxComputation class comment includes:

> This class is merely a crutch to get bounding box information without explicit metadata. This is likely to get deprecated following SMuFL support.

So as long as we can properly extract the bounding boxes from Bravura and Petaluma (and any other SMuFL font), then we should be good to go?

Maybe we need to eventually drop Gonville and replace it with Leland: https://github.com/MuseScoreFonts/Leland

It'd be nice to be 100% SMuFL.

Reply:
It should be possible to precompute the bounding boxes; the glyph cache I added makes a copy of the bounding box so that it only has to be computed once. So the trade off would be making the glyph data a little larger but removing the need to calculate the bounding box the first time a glyph is used.